spec/priorityQueue.js

```diff
@@ -0,0 +1,126 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import priorityQueue from '../src/priorityQueue'
+
+chai.use(chaiChange)
+
+describe('priorityQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(priorityQueue).to.be.a('function')
+  })
+
+  describe('enqueue()', () => {
+    it('pushes an element to the queue.', () => {
+      const queue = new priorityQueue()
+
+      expect(() => queue.enqueue('foo', '1'))
+        .to.alter(() => queue.length(), { from: 0, to: 1 })
+    })
+
+    it('pushes an element to the back of the queue.', () => {
+      const queue = new priorityQueue()
+      queue.enqueue('foo', 1)
+
+      expect(queue.back())
+        .to.eql({entry: 'foo', priority: 1})
+    })
+  })
+
+  describe('dequeue()', ()=> {
+    it('returns and removes the front element in the queue.', () => {
+      const queue = new priorityQueue()
+      queue.enqueue('foo')
+
```

Comment:
For a slightly better test, I'd add more than one element to the `queue()` you're using in your test. If you have more than one element, then you can ensure that you're dequeuing from the front of the queue and not the back of the queue.